force-app/main/default/lwc/platformDevCertCalculator/platformDevCertCalculator.html

```diff
@@ -0,0 +1,106 @@
+<template> 
+    <lightning-card title="PD1 Calculator">
+        <lightning-button 
```

Comment:
Kinda advanced but try and create an automated way to calculate the scores when the inputs have changes